rerun_js/web-viewer/build-wasm.mjs

```diff
@@ -31,13 +31,9 @@ function wasm(mode) {
   }
 }
 
-child_process.execSync(
-  "cargo run -p re_dev_tools -- build-web-viewer --debug --target no-modules-base -o rerun_js/web-viewer",
-  { cwd: __dirname, stdio: "inherit" },
-);
```

Comment:
I'm confused by what these lines were doing previously.

Reply:
It was redundant, remainder here after I moved it to a function with copy instead of cut :smile: